app/components/profile-form.tsx

```diff
@@ -0,0 +1,240 @@
+"use client";
```

Comment:
Rename to `my-profile-page.tsx` even if later we have to change the name because it's not the root profile page.

Reply:
done